torchao/float8/float8_utils.py

```diff
@@ -112,9 +110,12 @@ def tensor_to_amax(x: torch.Tensor, reduce_amax: bool = False) -> torch.Tensor:
 
 @torch.no_grad()
 def tensor_to_scale(
-    x: torch.Tensor, float8_dtype: torch.dtype, reduce_amax: bool = False
+    x: torch.Tensor,
+    float8_dtype: torch.dtype,
+    tile_size: Optional[Tuple[int, int]],
```

Comment:
default value?

Reply:
It would greatly reduce the the LOC of this PR but I have been wrestling with all our default values.

We we have like 3 abstraction layers where you can do casting with a mix of default and non default values. IMO I typically like have default values at the top most layer and then everything else requires args. This was my attempt to get us closer

Reply:
makes sense, maybe make tile_size keyword only to make the tensor wise case more readable?